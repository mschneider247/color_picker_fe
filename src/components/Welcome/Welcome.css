#welcomeTitle {
  padding-top: 6%;
  color: white;
  font-size: 2em;
  text-align: center;
  background-color: rgba(0, 0, 0, 0.6);
  width: 75%;
  height: 100px;
  border-radius: 10px;
}