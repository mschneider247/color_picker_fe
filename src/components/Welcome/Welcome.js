import React from 'react';
import './Welcome.css';

const Welcome = (props) => {
  
  return (
    <section>
      <h2>Welcome to Color_Picker!</h2>
    </section>
  )
}

export default Welcome;