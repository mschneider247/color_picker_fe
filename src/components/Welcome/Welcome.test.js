import React from 'react';
import { shallow } from 'enzyme';
import Welcome from './Welcome';

describe('Welcome component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Welcome/>)
  });

  it('Welcome should match its snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});