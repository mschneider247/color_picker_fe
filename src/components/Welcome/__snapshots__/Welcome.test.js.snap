// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component Welcome should match its snapshot 1`] = `
<section
  id="welcome"
>
  <h2
    id="welcomeTitle"
  >
    Welcome to Color_Picker!
  </h2>
</section>
`;
